index.js

```diff
@@ -0,0 +1,119 @@
+var days = [
+    'SUN',
+    'MON',
+    'TUE',
+    'WED',
+    'THU',
+    'FRI',
+    'SAT'
+];
+
+var months = [
+    'JAN',
+    'FEB',
+    'MAR',
+    'APR',
+    'MAY',
+    'JUN',
+    'JUL',
+    'AUG',
+    'SEP',
+    'OCT',
+    'NOV',
+    'DEC'
+];
+
+var monthdays = [31,28,31,30,31,30,31,31,30,31,30,31];
+
+var t = new Date();
+var tdate = t.getDate();
+var tday = t.getDay();
+var tmonth = t.getMonth();
+var tyear = t.getFullYear();  // 앞에 t 붙어있는거 -> 현재 날짜 가져옴.
+
+var d = tdate;
+var m = tmonth;
+var y = tyear;
+var nday = new Date(y,m,d).getDay();
+var arr = [];  // 날짜 저장
+
+function showDays(){
+
+    document.querySelector('.year').innerHTML = y;
+    document.querySelector('.dal').innerHTML = months[m];
+    document.querySelector('.date').innerHTML = d;
+    document.querySelector('.days').innerHTML = days[nday];
+
+    var lastday = monthdays[m];  // 이 달의 마지막 날
+    var firstday = new Date(y,m).getDay(); // 이 달의 첫번째 요일, 요일수만큼 0이 앞에 생긴다.
+    var count = "";
+
+    for (j=0; j<firstday; j++){
+        count += '<div class="zeros">&nbsp</div>';
+        document.querySelector('.numbers').innerHTML = count;
+        arr.push(0);
+    }
+
+    for (i=1; i<= lastday; i++){
+         if(i==tdate && m==tmonth && y==tyear){
```

Comment:
비교 연산자는 equality(==) 와 strict equality(===) 두가지가 있습니다.
어떤 차이가 있을까요?
> 참고자료
- [equality](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Equality)
- [strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)

또한 if문 안에 띄어쓰기도 변수 선언 할 때와 같이 한칸 띄워주시면 좋을 것 같습니다.
```suggestion
         if (i == tdate && m == tmonth && y == tyear){
```